_quarto.yml

```diff
@@ -0,0 +1,38 @@
+project:
+  type: website
+
+website:
+  title: "NFIDD"
+  site-url: https://nfidd.github.io/nfidd/
+  description: "Nowcasting and forecasting infectious disease dynamics"
+  image: images/logo.png
+  favicon: images/favicon-32x32.png
```

Comment:
take out whilst we don't have a logo?

```suggestion
```

Reply:
Sorry I missed this - I meant it just as a placeholder.